src/PowerPlaywright.Framework/Controls/Pcf/Classes/IDuration.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace PowerPlaywright.Framework.Controls.Pcf.Classes
+{
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Duration control class.
+    /// </summary>
+    public interface IDuration : IPcfControl
+    {
+        /// <summary>
+        /// Sets the value of the Duration control.
+        /// </summary>
+        /// <param name="value">The value.</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        Task SetValueAsync(string value);
```

Comment:
If we're going to keep this as as a string we may want to document the acceptable formats e.g. `x minute(s)/hour(s)/day(s)`. Alternatively we accept only integers and state that the unit is minutes.